app/assets/javascripts/common/password_validation.js

```diff
@@ -39,15 +30,41 @@ PasswordValidation = function() {
         "X-CSRF-Token": token
       },
       data: {"password": password}
-    }).done(function(data, status){
-      console.log(status)
-      console.log(data)
-      this.timer = null;
+    }).done((data, status) => {
+      let response;
       if (status === "success") {
-        this.response = data["errors"];
+        response = data["errors"];
       } else {
-        this.response = null;
+        // if we don't get a response, mark all validations as failures
+        response = Validations;
       }
+      this.updateChecklist(response);
+      this.updateSubmitStatus(response);
```

Comment:
...and here, we are actually passing in an array of errors _from_ the response, not the response itself. Would be good to name this `errors` to be explicit about what data we are working with.

Reply:
Done.